src/main/java/hudson/tasks/junit/JUnitParser.java

```diff
@@ -66,16 +67,20 @@ public JUnitParser() {
     public JUnitParser(boolean keepLongStdio) {
         this.keepLongStdio = keepLongStdio;
         this.allowEmptyResults = false;
+        this.makeUnstable = true;
+    
     }
 
     /**
      * @param keepLongStdio if true, retain a suite's complete stdout/stderr even if this is huge and the suite passed
      * @param allowEmptyResults if true, empty results are allowed
+     * @param makeUnstable if true, build will be Unstable if there are any failed test cases
      * @since 1.10
      */
-    public JUnitParser(boolean keepLongStdio, boolean allowEmptyResults) {
+    public JUnitParser(boolean keepLongStdio, boolean allowEmptyResults, boolean makeUnstable) {
```

Comment:
It breaks the binary compatibility, a new constructor should be created

Reply:
New constructor is created.